packages/module/src/MultiContentCard/MultiContentCard.tsx

```diff
@@ -1,3 +1,5 @@
+/** @jsx jsx */
```

Comment:
I think there should be a way of getting rid of these annotations

Reply:
@fhlavac We get test failures without that line. See: https://emotion.sh/docs/css-prop#jsx-pragma